src/util/ringct.rs

```diff
@@ -172,16 +175,17 @@ impl EcdhInfo {
         )
     }
 
-    /// TODO
+    /// Opens the commitment and verifies it against the actual commitment present within the transaction.
```

Comment:
:upside_down_face: Since this API is public, the caller is actually free to pass in any commitment that they want to compare against, not just one present in the transaction.

Reply:
Good point, will update naming accordingly!